### ISSUE TITLE
Demo on webpage does not work

### ISSUE THREAD

**Issue by dphizler** [ratchetphp/Ratchet#356]
Here is the url:
http://socketo.me/demo

**Comment by cboden** [comment]
Can you elaborate? I just tested and it's working for me.

**Comment by dphizler** [comment]
Hi,

Sorry, I assumed it was a generalized problem.

1. I go to the demo page (Google chrome), enter my name
[image: Inline image 1]

2. As can be seen in the console, I get a "Unexpected response code: 301
[image: Inline image 2]

I hope that helps, I don't think I can give you more information. Keep in
mind that I managed to make the Ratchet package work on my local machine
with the test tutorial.

On a side note, I tried to do the Push integration with the zmq dll and all
and I get this error when I run the "push-server.php" on my local machine:

--------------------------------------------------------------------
C:\wamp\www\ratchet>php bin/push-server.php

Warning: stream_select(): cannot represent a stream of type ZMQ_FD as a
select()able descriptor in C:\wamp\www\ratchet\vendor
\react\event-loop\StreamSelectLoop.php on line 255
--------------------------------------------------------------------

From what I can tell, I need to have a 32 bit wamp server installed for it
to work.

Thank you,
Phil
